phpstan.neon.dist

```diff
@@ -20,14 +20,10 @@ parameters:
 		- tests/Fixtures/app/var/
 		- tests/Fixtures/Symfony/Maker
 		# BC layer
-		- src/deprecated_interfaces.php
 		- tests/Fixtures/TestBundle/BrowserKit/Client.php
 		# The Symfony Configuration API isn't good enough to be analysed
 		- src/Symfony/Bundle/DependencyInjection/Configuration.php
-		# Phpstan runs on phpunit > 9, a signature changed in this file
-		- src/Symfony/Bundle/Test/Constraint/ArraySubsetLegacy.php
 		# Imported code (temporary)
-		- src/Symfony/Bundle/Test/BrowserKitAssertionsTrait.php
 		- tests/Symfony/Bundle/Test/WebTestCaseTest.php
 		- tests/Fixtures/TestBundle/Security/AbstractSecurityUser.php
```

Comment:
This line needs to be checked

Reply:
Same for this. Can the BC layer be removed?

Reply:
Yes, can you do it in another PR?

Reply:
https://github.com/api-platform/core/pull/4867